ghost/i18n/locales/fr/portal.json

```diff
@@ -39,11 +39,11 @@
     "Continue": "Continuer",
     "Continue subscription": "Continuer l'abonnement",
     "Could not sign in. Login link expired.": "Impossible de se connecter. Le lien de connexion a expiré.",
-    "Could not update email! Invalid link.": "Impossible de mettre à jour l'e-mail ! Lien invalide.",
+    "Could not update email! Invalid link.": "Impossible de mettre à jour l'e-mail\u00a0! Lien invalide.",
```

Comment:
Did the translator put some weird space encoding in here or something? 😝

Reply:
oh dam weird, must've been linting that did that??

Will unstage that file. 🤣

Reply:
oh dam weird, must've been linting that did that??

Will unstage that file. 🤣